consumer/processing.go

```diff
@@ -161,6 +161,28 @@ func writeRecommendations(
 	return tStored, nil
 }
 
+func (consumer *KafkaConsumer) writeInfoReport(
+	msg *sarama.ConsumerMessage, message incomingMessage, infoStoredAtTime time.Time,
+) error {
+	err := consumer.Storage.WriteReportInfoForCluster(
+		*message.Organization,
+		*message.ClusterName,
+		message.ParsedInfo,
+		infoStoredAtTime,
+	)
+	if err != nil {
```

Comment:
this is redundant https://go.dev/play/p/d2sRXgyH2ds you can compare nil and an error, maybe there's a way to leave gocyclo unchanged if we solve this?

Reply:
Maybe it will look cleaner, but it will still need to increase the gocyclo because the function with 10 gocyclo points is the `ParseMessage` one, and not this one